src/app/server/JointFabricDatastore.cpp

```diff
@@ -561,7 +628,7 @@ JointFabricDatastore::RemoveBindingFromEndpointForNode(uint16_t listId, NodeId n
 }
 
 CHIP_ERROR JointFabricDatastore::UpdateACLEntry(
-    Clusters::JointFabricDatastore::Structs::DatastoreACLEntryStruct::Type & entryToUpdate,
```

Comment:
I don't think we actually need this command. There is an addACLEntry and a removeACLEntry. There is no updateACLEntry so this is not logic we need to manage

Reply:
Agreed and removed.